unity-renderer/Assets/Scripts/MainScripts/DCL/Analytics/SupportAnalytics/SupportAnalytics.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace SupportAnalytics
```

Comment:
I suggest to be it just in `Analytics` namespace for simplicity

Reply:
All analytics have specific classes

Reply:
classes are specific, but namespace is an application subset. From my point of view, there is no need make every folder as a namespace. You can disable it in Rider by right-clicking on folder and selecting Properties. Then in popup you can disable respective toggle
![image](https://github.com/decentraland/unity-renderer/assets/35366872/c5ca296a-78bb-4c81-b12c-fe828c52eb04)

Reply:
sorry missread the comment, thought you were referring to the classname, gonna remove now the namespace